src/ViewLayer/SongPlayerView/SongControl/SongControlView.cpp

```diff
@@ -0,0 +1,81 @@
+#include "SongControlView.h"
+#include "I_SongControlPresenter.h"
+#include "I_SongControlUi.h"
+#include "I_SongPlayerState.h"
+#include <QPushButton>
+#include <QSlider>
+SongControlView::SongControlView
+(
+    I_SongControlPresenter& songControlPresenter,
+    I_SongControlUi& songControlUi
+):
+    songControlPresenter_(songControlPresenter),
+    songControlUi_(songControlUi),
+    songState_(songControlPresenter_.songPlayerState())
+{
+    //Connecting buttons to data
```

Comment:
Remove Comment

Reply:
I don't completely agree with the removal of comments, but I will remove them.